query_execution/CMakeLists.txt

```diff
@@ -33,6 +33,9 @@ if (ENABLE_DISTRIBUTED)
   add_library(quickstep_queryexecution_BlockLocator BlockLocator.cpp BlockLocator.hpp)
 endif(ENABLE_DISTRIBUTED)
 add_library(quickstep_queryexecution_ForemanBase ../empty_src.cpp ForemanBase.hpp)
+if (ENABLE_DISTRIBUTED)
+  add_library(quickstep_queryexecution_ForemanDistributed ForemanDistributed.cpp ForemanDistributed.hpp)
+endif(ENABLE_DISTRIBUTED)
```

Comment:
This is consistent in almost every, if not all, `CMakeLists.txt`s that `if (xxx)` has a whitespace , while all else, including `elseif(xxx)` and `else(xxx)`, do not have whitespace in between.